draft-ietf-rats-architecture.md

```diff
@@ -360,6 +363,42 @@ are signed.  For example, values might have been generated at boot, and then
 used in claims as long as the signer can guarantee that they cannot have changed
 since boot.
 
+The term freshness is applied to both Evidence and Attestation Results in
+this document. Unfortunately, the term freshness is also a bit overloaded and
+in order to illustrate the application of the term in this document better,
+two characteristics that can be part of freshness are elaborated on below:
+recentness and uniqueness:
+
+Recentness:
+
+: a quality of Evidence that is based on the time that has passed between
+creation of the evidence by an Attester (typically by "sealing" a Claims set
+via signing) and the reception of that Evidence by a Verifier or (relaying
+party).
+
+: Evidence can be not recent enough, if the Evidence created from collected
+Claims is too old. A nonce supplied during a Challenge/Response handshake and
+a time-stamp based proof-of-recentness between time-synchronized peers are
+both usage scenarios in which recentness characteristics are provided to the
+Verifier.
+
+Uniqueness:
```

Comment:
I don't understand why this discussion is needed/relevant.   (I might understand it in the context of a specific protocol/mechanism, but I don't understand why it belongs in the architecture.)

Reply:
As various individuals seem to have 2+ points of view on this, I'd assume this will create a severe caveat, if not illustrated and unambiguously resolved in the architecture I-D . In the end, if we do not agree on this intuitively, how is the architecture doc not in charge of preventing this - obviously now identified - cause of confusion to other readers?

Reply:
I think the uniqueness property is addressed by the credential used to authenticate the Attester to the Verifier (and subsequently used to integrity protect the Evidence messages). If the credential uniquely identifies the Attester then the Evidence is unique to the session to which the authentication applies. 
If the credential is a group credential (e.g. EPID / DAA) then the uniqueness is to the members of the group. We have other sections on Privacy and authentication of the endpoint. (Or maybe I'm wrong about the latter). But it might make sense to have a section on "Authentication of the Endpoint" as the way to talk about security properties of uniqueness / non-uniqueness?